packages/lib-utils/src/k8s/hooks/useK8sWatchResource.ts

```diff
@@ -30,7 +30,7 @@ const NOT_A_VALUE = '__not-a-value__';
  * ```
  */
 export const useK8sWatchResource = <R extends K8sResourceCommon | K8sResourceCommon[]>(
-  initResource: WatchK8sResource | null,
```

Comment:
This is explicitly needed if the `initResource` is dynamically calculated. 

I created a PR to specifically add it in: https://github.com/openshift/dynamic-plugin-sdk/pull/51

Reply:
Thanks @andrewballantyne! Thank you for linking your PR as well. I mistakenly took this for a different change I had introduced in my PR. 
Reverted 👍